app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayoutViewModel.kt

```diff
@@ -99,7 +98,8 @@ class OmnibarLayoutViewModel @Inject constructor(
         val updateOmnibarText: Boolean = false,
         val shouldMoveCaretToEnd: Boolean = false,
         val shouldMoveCaretToStart: Boolean = false,
-        val tabs: List<TabEntity> = emptyList(),
```

Comment:
This list was only used to count the tabs. Each tab would contain a list of all these entities and with lots of tabs this would unnecessarily take up a lot of memory.